svd-parser/src/lib.rs

```diff
@@ -171,6 +199,9 @@ mod registerproperties;
 mod usage;
 mod writeconstraint;
 
+#[cfg(feature = "expand")]
+mod expand;
+
```

Comment:
maybe we should expose the function as well?

```suggestion
#[cfg(feature = "expand")]
mod expand;

#[cfg(feature = "expand")]
pub use expand::expand;
```

Reply:
Thanks